src/components/MultiLink/MultiLink.vue

```diff
@@ -245,7 +245,27 @@ export default Vue.extend({
     },
 
     nodeTranslate(node: Node): string {
-      return `translate(${node.x || 0}, ${node.y || 0})`;
+      let forcedX = node.x || 0;
+      let forcedY = node.y || 0;
+
+      const svgEdgePadding = 5;
+
+      const minimumX = svgEdgePadding;
+      const minimumY = svgEdgePadding;
+      const maximumX = this.svgDimensions.width - this.markerSize - svgEdgePadding;
+      const maximumY = this.svgDimensions.height - this.markerSize - svgEdgePadding;
+
+      forcedX = forcedX < minimumX ? minimumX : forcedX;
+      forcedX = forcedX > maximumX ? maximumX : forcedX;
+      forcedY = forcedY < minimumY ? minimumY : forcedY;
+      forcedY = forcedY > maximumY ? maximumY : forcedY;
```

Comment:
I actually think if statements would be more readable here, since the else clause is to "do nothing".

Reply:
Why go through the `forced{X|Y}` variables? The code would be shorter if you directly set `node.x` and `node.y`, then use `node.x` and `node.y` in line 268 as well.

Reply:
From the PR description:

>I found that updating node.x and node.y and then doing translate(node.x, node.y) didn't work. I think that's because the simulation updates node.x and node.y before the node is drawn, so there's a bit of a race conditions. My solution was to set the translate to the forced values and assign those values back to node.x and node.y. That seems to keep everything settled.

Reply:
Ok. This requires a comment in the code, so future versions of ourselves don't wonder anew what's happening here.

Reply:
Added a comment and addressed your if statement preference in 8299a9f